src/control/server/mgmt_system.go

```diff
@@ -1072,6 +1073,148 @@ func (svc *mgmtSvc) SystemExclude(ctx context.Context, req *mgmtpb.SystemExclude
 	return resp, nil
 }
 
+func (svc *mgmtSvc) SystemDrain(ctx context.Context, req *mgmtpb.SystemDrainReq) (*mgmtpb.SystemDrainResp, error) {
```

Comment:
Needs a documentation comment

Reply:
Will fix in subsequent PR for SystemReint if that's okay